app/js/templates/editmodal.hbs

```diff
@@ -27,10 +27,14 @@
         {{/each}}
         <option value="{{defaultCategory.id}}" selected>{{defaultCategory.name}}</option>
     </select>
+
     <label class="editUserData">Allow retake (tick to allow):</label>
     <input id="canRetake" type="checkbox" {{#if user.canRetake}}checked{{/if}}>
 
-    <div class="message-target">
+    <label class="editUserData">Allow to finish the auto-completed test (tick to allow):</label>
```

Comment:
this should only show if the users test has auto-completed, it currently shows for all users